Formula/kubie.rb

```diff
@@ -0,0 +1,17 @@
+class Kubie < Formula
+  desc "Much more powerful alternative to kubectx and kubens"
+  homepage "https://blog.sbstp.ca/introducing-kubie/"
+  url "https://github.com/sbstp/kubie/archive/v0.7.1.tar.gz"
+  sha256 "6e897e6b4877a5fbed8721bee2469c15c1a7846b966bbfae8d43e12e5c0ea0c1"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+    bash_completion.install "./completion/kubie.bash"
+  end
+
+  test do
+    system "#{bin}/kubie", "--version"
```

Comment:
can you improve the test block?

Reply:
@chenrui333 thanks for the feedback.

I have added an assertion for the version that comes back from `kubie --version`.

I have poked around some of the other formulas and many of them are just smoke tests like this one to test that the compilation has produced a valid binary and that it can be called.

I'm not sure how much more testing can be done here that wouldn't be testing the actual software but if you have suggestions I'm happy to hear them.